skorch/tests/callbacks/test_training.py

```diff
@@ -1083,3 +1083,11 @@ def initialize_module(self, *args, **kwargs):
 
         assert save_params_mock.call_count == 1
         save_params_mock.assert_has_calls([call(f_mymodule='train_end_mymodule.pt')])
+
+
+class TestNoCallbacks(TestCheckpoint):
+    def test_no_callbacks(self, net_cls, data):
```

Comment:
Please move this test into class `TestNeuralNet` inside `tests/test_net.py` (just this test, no extra class required).